app/Kconfig

```
@@ -560,6 +560,11 @@ endif
 #Logging
 endmenu
 
+config ZMK_SETTINGS_RESET_ON_START
+    bool "Delete all persistent settings when the keyboard boots"
+    default n
+    select SETTINGS
```

I wonder if instead this should be:

```suggestion
    depends on SETTINGS
```

I don't think it makes much sense to turn this on unless you've already got other settings turned on elsewhere?